df-valve/src/main/java/com/hashmapinc/tempus/witsml/valve/dot/DotValve.java

```diff
@@ -18,14 +18,18 @@
 import java.util.logging.Logger;
```

Comment:
Added query logic directly in the `getObject` method with no object type checking.

I will move ALL this logic to the `Delegator` when I support `WELLBORE` querying.

I know this is dumb, but please just trust me for now. This is the result of my previous bad choices, my love for killing waffle cards, and some important PR's that are pending approval.

For more info, please just message me. But trust me, this is for the best for now.